buildroot-external/package/hailo8-firmware/hailo8-firmware.mk

```diff
@@ -13,9 +13,13 @@ define HAILO8_FIRMWARE_EXTRACT_CMDS
 	cp $(HAILO8_FIRMWARE_DL_DIR)/$(HAILO8_FIRMWARE_SOURCE) $(@D)
 endef
 
+define HAILO8_FIRMWARE_BUILD_CMDS
+	cp $(@D)/$(HAILO8_FIRMWARE_SOURCE) $(@D)/hailo8_fw.bin
+endef
+
```

Comment:
Is that directly really related to the driver code move in some ways? If not, I think a separate PR would be better.

Reply:
Yes, already pointed out here: https://github.com/home-assistant/operating-system/pull/3721#issuecomment-2523179200

Without this change the PR will break the upstream driver.

Reply:
Ah I see. Make sense then to have this in this PR!